pkg/reader/ccip.go

```diff
@@ -972,6 +978,64 @@ func (r *ccipChainReader) getOffRampSourceChainsConfig(
 	return res, nil
 }
 
+// selectorsAndConfigs wraps the return values from getAllSourceChainConfigs.
+type selectorsAndConfigs struct {
+	Selectors          []uint64            `mapstructure:"F0"`
```

Comment:
I believe we don't need to use `mapstructure` here. check this [example](https://github.com/smartcontractkit/chainlink-ccip/blob/ad2992ddf2377fe55c65e1aca9cb03c1c8cd8b2a/internal/reader/home_chain.go#L404-L407) where it's just using normal tag

Reply:
I believe it's actually the json tags in that example which aren't needed. When a tag isn't provided the field name is used.